data/xml/2020.paclic.xml

```diff
@@ -399,14 +399,14 @@
       <url hash="ff8e9d81">2020.paclic-1.46</url>
     </paper>
     <paper id="47">
-      <title>Exploring Discourse of Same-sex Marriage in <fixed-case>T</fixed-case>aiwan: A Case Study of Near-Synonym of <fixed-case>HOMOSEXUAL</fixed-case> in Opposing Stances</title>
+      <title>Exploring Discourse on Same-sex Marriage in <fixed-case>T</fixed-case>aiwan: A Case Study of Near-Synonym of <fixed-case>HOMOSEXUAL</fixed-case> in Opposing Stances</title>
       <author><first>Han-Tang</first><last>Hung</last></author>
       <author><first>Shu-Kai</first><last>Hsieh</last></author>
       <pages>411–419</pages>
       <url hash="549ab4b1">2020.paclic-1.47</url>
     </paper>
     <paper id="48">
-      <title>A simple and efficient ensemble classifier combining multiple neural network models on social media datasets in <fixed-case>V</fixed-case>ietnamese</title>
+      <titleA Simple and Efficient Ensemble Classifier Combining Multiple Neural Network Models on Social Media Datasets in <fixed-case>V</fixed-case>ietnamese</title>
```

Comment:
You've introduced an error here that broke the build.

Reply:
Oh I am sorry, I am correcting that.